2021-06-22 22:34:06,027 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:37:02,104 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:37:05,861 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:38:51,417 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:38:54,631 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:39:26,752 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:39:33,224 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:40:04,967 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 22:40:25,319 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.OperationalError) (1049, "Unknown database 'devops-cicd-api'")
(Background on this error at: http://sqlalche.me/e/14/e3q8)
2021-06-22 23:09:03,126 - WARNING - view.py - cicdJobsInsert - 102 - cicd data add failure  Exception(pymysql.err.IntegrityError) (1048, "Column 'env' cannot be null")
[SQL: INSERT INTO cicdmg (env, appname, appversion, branch, instance_ip, giturl, language_type, release_type, release_reason, jenkins_callback, releasetime) VALUES (%(env)s, %(appname)s, %(appversion)s, %(branch)s, %(instance_ip)s, %(giturl)s, %(language_type)s, %(release_type)s, %(release_reason)s, %(jenkins_callback)s, %(releasetime)s)]
[parameters: {'env': None, 'appname': 'op-service-abc', 'appversion': '1.1.1.0', 'branch': 'release', 'instance_ip': '1,.,1,.,1,.,1', 'giturl': "'https://github.com/breaklinux/nginxWebUI.git", 'language_type': 'VM', 'release_type': 'vm', 'release_reason': 'op-test', 'jenkins_callback': 'http://192.168.1.200:8080/blue/organizations/jenkins/op-service-abc/detail/op-service-abc/1', 'releasetime': datetime.datetime(2021, 6, 22, 23, 8, 57, 523408)}]
(Background on this error at: http://sqlalche.me/e/14/gkpj)
